### PR TITLE
Ridge classifier and test cases

### PR DESCRIPTION
**Pull request by Arneja12**
Partial #21

### CODE REVIEW

File: datascientist/model/classification/skl/linear_model/tests/test_ridgeclassifier.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    metrics = 'accuracy_score'
+    answer = _ridgeclassifier(train=(x_train, y_train), test=(x_test, y_test), metrics=metrics)
+    assert answer[0] == 'RidgeClassifier'
+    assert answer[1] == 1.0
+    assert answer[2] is None
+
+    answer = _ridgeclassifier(train=(x_train, y_train), test=(x_test, y_test), metrics=metrics, x_predict=x_test)
+    assert arr == any([0, 1, 0, 0])
+
```
Comment (sidhu1012) [inline datascientist/model/classification/skl/linear_model/tests/test_ridgeclassifier.py:33]: Remove it

File: datascientist/model/classification/skl/linear_model/ridgeclassifier.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        accuracy = jaccard_score(test[1], y_hat)
+    if metrics == 'accuracy_score':
+        accuracy = accuracy_score(test[1], y_hat)
+
+    if x_predict is None:
+        return (model_name, accuracy, None)
+
+    y_predict = model.predict(x_predict)
+    return (model_name, accuracy, y_predict)
+
```
Comment (sidhu1012) [inline datascientist/model/classification/skl/linear_model/ridgeclassifier.py:33]: Remove it

File: datascientist/model/classification/skl/linear_model/tests/test_ridgeclassifier.py (hunk 1 above)
Comment (Arneja12) [reply datascientist/model/classification/skl/linear_model/tests/test_ridgeclassifier.py]: Done

File: datascientist/model/classification/skl/linear_model/ridgeclassifier.py (hunk 2 above)
Comment (Arneja12) [reply datascientist/model/classification/skl/linear_model/ridgeclassifier.py]: Done

**Pull request merged**